frontend/javascripts/admin/dataset/dataset_upload_view.tsx

```diff
@@ -681,9 +724,72 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
             />
           ) : null}
 
+          {this.state.ongoingUploads.length > 0 && (
+            <Alert
+              message={
+                <>
+                  Ongoing unfinished Dataset Uploads{" "}
```

Comment:
I think I would omit the word "ongoing" because it could be interpreted as if the upload was still progressing. However, this is likely not the case.